Controller/DbControllers/PlaylistSongController.cs

```diff
@@ -32,5 +33,20 @@ public void addSongToPlaylist(int songID, int playlistID)
 
             
         }
+
+        public List<Song> GetSongsFromPlaylist(int playlistID)
+        {
+            List<PlaylistSong> playlistSongs = _context.PlaylistSongs.Where(ps => ps.PlaylistID == playlistID).ToList();
+
+            List<Song> songs = new List<Song>();
+            foreach (var playlistSong in playlistSongs)
+            {
+                var id = playlistSong.SongID;
+                var wou = "sfl";
```

Comment:
Wat is dit?

Reply:
stukje debugging vergeten wel te halen :)